<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>715</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: rgb(93, 123, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.891626, y2:0.902, stop:0 rgba(43, 43, 43, 255), stop:1 rgba(194, 194, 194, 255));
}
</string>
   </property>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;; 
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Vision System Fabridoce Machine 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cam1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Camera 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cam2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Camera 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_box1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>512</width>
      <height>384</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_box2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>512</width>
      <height>384</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="manual_trigger_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual Trigger</string>
    </property>
   </widget>
   <widget class="QToolButton" name="simButton">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>560</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SaveBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>610</y>
      <width>77</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Img</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="timelabel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>520</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time: 0.0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
